ts/data/requisition.ts

```diff
@@ -1,4 +1,11 @@
-import { CellStatus, makeIcon, makeNameDiv } from "../util/html-utils";
+// Informatics Review Table
```

Comment:
This comment might be misleading here as the file includes table definitions for 2 other tables as well

Reply:
Removed